openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -265,7 +311,27 @@
                     percircleOptions.progressBarColor = data.colors[data.trace_order.indexOf(key)];
                 }
                 percircles.push(percircleOptions);
+                if(data.trace_order !== undefined) {
```

Comment:
same here regarding `data.trace_order !== undefined` and `adaptiveFilterSummary(data, percircles)`.